content/en/blog/2020/new-deployment-model/index.md

```diff
@@ -0,0 +1,67 @@
+
+{{< image width="100%"
+    link="single-cluster.svg"
+    alt="Istio mesh in a single cluster"
+    title="Single cluster Istio mesh"
+    caption="Istio mesh in a single cluster"
+    >}}
+
+With this new deployment model in Istio 1.7, it is possible to run Istiod on a separate cluster (the `control plane cluster`) from services, as shown in the diagram below. The `control plane cluster` is owned by mesh operator while the mesh admin owns the cluster that has services deployed in the mesh. Mesh admin has no access to the `control plane cluster`. If you are interested in exploring this, you can follow the [central istiod single cluster step by step guide](https://github.com/istio/istio/wiki/Central-Istiod-single-cluster-steps).
+
+{{< image width="100%"
+    link="single-cluster-central-Istiod.svg"
+    alt="Istio mesh in a single cluster with Istiod outside"
+    title="Single cluster Istio mesh with Istiod outside"
+    caption="Single cluster Istio mesh with Istiod in a control plane cluster"
+    >}}
+
+Mesh admins can expand the service mesh to multiple clusters, which are managed by the same Istiod running in the `control plane cluster`. The `primary data plane cluster` also serves as the Istio configuration cluster, in addition to run services. The `control plane cluster` reads Istio configurations from the `primary data plane cluster` and pushes them to all of the data plane clusters per the diagram below.
+
+{{< image width="100%"
+    link="multiple-clusters-central-Istiod.svg"
+    title="Multicluster Istio mesh with Istiod outside"
+    caption="Multicluster Istio mesh with Istiod in a control plane cluster"
+    >}}
+
+Mesh operators may further expand this deployment model to manage multiple Istio control planes from a control plane cluster that runs multiple Istiod control planes:
+
+{{< image width="100%"
+    link="multiple-central-Istiods.svg"
+    alt="Istio meshes in single clusters with Istiod outside"
+    title="Multiple single clusters Istio meshes with Istiod outside"
+    caption="Multiple single clusters with multiple Istiod control planes in a control plane cluster"
+    >}}
+
+The `control plane cluster` can be used to host multiple Istiod control planes and each Istiod manages its own `data plane cluster`. Mesh operators could install another Istio mesh in the `control plane cluster` and configure its `istio-ingress` gateway to route traffic from a data plane cluster to its corresponding Istiod control plane.
+
+## Conclusion
+
+This new deployment model enables the Istio control plane to be run and managed by mesh operators who have operational expertise in Istio. Mesh operators can run the control plane in their own control plane clusters, provide it as a service to mesh admins. Mesh operators can optionally run multiple Istiod control planes in the control plane cluster, deploy their own Istio mesh and use their `istio-ingress` gateway to control access to these Istiod control planes.
```

Comment:
We should make it clear that mesh operators can run the control planes however they want (for example they could run them on VMs), they can run multiple per cluster, etc. That is their implementation choice, we provide some examples to get started but there is lots of flexibility.

Reply:
Thanks Sven!  Updated PTAL